features/campaigns/efforts.html

```diff
@@ -26,12 +26,12 @@
 <hr/>
 
 <div class="row subfeature">
-  <div class="col-md-6 order-last order-sm-12">
```

Comment:
Did `order-sm-12` caused a different layout on small screens? We don't need it anymore?

Reply:
According to the migration guide, `order-*` classes above 6 have been removed. I tested on mobile and this didn't seem to have any effect.